src/parser/jobs/pld/changelog.tsx

```diff
@@ -2,6 +2,13 @@ import CONTRIBUTORS from 'data/CONTRIBUTORS'
 import React from 'react'
 
 export const changelog = [
+	{
+		date: new Date('2022-10-01'),
+		Changes: () => <>
+			Updated supported patches to include 6.2.
+		</>,
+		contributors: [CONTRIBUTORS.STYRFIRE],
+	},
```

Comment:
Since nothing functional has changed, I don't think we do changelogs on just patch bumps anymore – the fact that it's not greyed out in the selection menu should be notification enough.